blog/views.py

```diff
@@ -1,40 +1,41 @@
-from django.shortcuts import render, get_object_or_404
+from django.shortcuts import render, redirect, get_object_or_404
```

Comment:
You forgot to import the `redirect` shortcut, which was throwing an error when you tried to call it here: `return redirect('blog.views.post_detail', pk=post.pk)`